cartographer_rviz/cartographer_rviz/submaps_display.cc

```diff
@@ -149,6 +164,51 @@ void SubmapsDisplay::update(const float wall_dt, const float ros_dt) {
   }
 }
 
+void SubmapsDisplay::AllEnabledChanged() {
+  ::cartographer::common::MutexLocker locker(&mutex_);
+  const bool visibility = visibility_all_enabled_->getBool();
+  for (auto& trajectory : trajectories_) {
+    for (auto& submap : trajectory) {
+      DisconnectVisibilityChange(submap);
+      submap->set_visibility(visibility);
+      ConnectVisibilityChange(submap);
+    }
+  }
+}
+
+void SubmapsDisplay::VisibilityChanged(DrawableSubmap* const submap) {
+  ::cartographer::common::MutexLocker locker(&mutex_);
+  // Handle the case when the user was holding Ctrl,
```

Comment:
please remove the CTRL convenience. I think the whole function can then go away.